package.json

```diff
@@ -27,18 +27,6 @@
     "@babel/traverse": "^7.0.0",
     "@babel/types": "^7.0.0",
     "ansi-to-html": "^0.6.4",
-    "babel-code-frame": "^6.26.0",
-    "babel-core": "^6.25.0",
-    "babel-generator": "^6.25.0",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.0",
-    "babel-plugin-transform-flow-strip-types": "^6.22.0",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-plugin-transform-react-jsx": "^6.24.1",
-    "babel-preset-env": "^1.7.0",
-    "babel-template": "^6.26.0",
-    "babel-traverse": "^6.26.0",
-    "babel-types": "^6.26.0",
-    "babylon": "^6.17.4",
     "babylon-walk": "^1.0.2",
```

Comment:
Should we remove this, or do a PR to their branch to update to `@babel/types` instead of `babel-types`

It might cause issues very rarely, but still might occur

Reply:
worth doing some perf testing. babylon-walk was WAY faster than babel-traverse for some cases where we don't need scope tracking and things.

Reply:
I'm pretty sure it will still be slower, haven't tested it. I did however open up a PR to babylon-walk https://github.com/pugjs/babylon-walk/pull/2